portfolio/src/main/webapp/task.js

```diff
@@ -174,3 +174,7 @@ function validateTaskDuration(duration) {
     return {isValid: true, errorMessage: null};
   }
 }
+
+module.exports.getDurationMinutes = getDurationMinutes;
```

Comment:
Consider creating a Javascript class for duration. I think Date works for instant, but I don't see a counter-part for Duration Java class.

To begin with, the Duration class can just be instantiated with milliseconds via constructor. And you can start using the Duration class in the code-base.

But if you want to handle this later on in the internship that's fine too. Consider adding TODO comment.